docs/NooBaaNonContainerized/Health.md

```diff
@@ -0,0 +1,350 @@
+# NooBaa Non Containerized - Health CLI
+
+1. [Introduction](#introduction)
+2. [Installation](#installation)
+3. [General Information](#general-information)
+5. [Usage](#usage)
+6. [Output](#output)
+7. [Example](#example)
+8. [Health Errors](#health-errors)
+    1. [General Errors](#general-errors)
+    2. [Accounts / Buckets Health Errors](#accounts--buckets-health-errors)
+
+
+## Introduction
+The Health CLI tool is a powerful command-line interface designed to efficiently analyze the NooBaa service, endpoints, accounts and buckets health.
+
+## Installation
+Health CLI installed automatically during the NooBaa RPM installation.
+For more details about NooBaa RPM installation, see - [Getting Started](./GettingStarted.md).
+
+
+## General Information 
+* NooBaa Health CLI will analyze the following aspects - 
+  - `NooBaa Service health`
+    - Verifying that the NooBaa service is active and has a valid PID (not equal to 0).
+  - `NooBaa endpoints health`
+    - Ensuring that the NooBaa endpoint is operational (running).
+  - `NooBaa Accounts Health`
+    - Iterating accounts under the config directory.
+    - Confirming the existence of the account's configuration file and its validity as a JSON file.
+    - Verifying that the account's new_buckets_path is defined and allow_bucket_creation is set to true.
```

Comment:
I think it is easier to read with code inline:
```suggestion
    - Verifying that the account's `new_buckets_path` is defined and `allow_bucket_creation` is set to true.
```

Question: What if the user sets the new_buckets_path and explicitly set allow_bucket_creation to false?